homeassistant/components/deconz/climate.py

```diff
@@ -51,6 +52,11 @@
     HVAC_MODE_OFF: "off",
 }
 
+HVAC_MODE_TO_DECONZ_SPZB0001 = {
+    HVAC_MODE_HEAT: "auto",
+    HVAC_MODE_OFF: "off",
+}
```

Comment:
```suggestion
```

Reply:
Sorry for wasting your time, I misunderstood you with the global variable. now it is as expected